### DISCUSSION TITLE
About recent ColorChecker formulations and commercial usage.

### DISCUSSION THREAD

**Discussion by peteroupc** [colour-science/colour#706] [General]
X-Rite changed the formulation of the colors used to produce ColorChecker charts since November 2014; see the [2015 announcement](http://xritephoto.com/ph_product_overview.aspx?ID=938&Action=Support&SupportID=5884) for details.

However, based on what I see at the end of [this article from 2010](http://xritephoto.com/ph_product_overview.aspx?ID=1257&Action=Support&SupportID=5159), which describes the old ColorChecker colors, the 2015 announcement does not clarify whether the data given there allows commercial use, and neither is it clear whether the "COLORCHECKER_2005" data in this repository was permitted to be included in this project (which has a free software license that allows commercial as well as noncommercial use). (I'm not a lawyer.)

**Comment by thomasmansencal** [comment]
Hi @peteroupc,

Very good point, the data in Colour is coming from [1]. Taking a look at the latest version of the spreadsheet, there is no licensing information. Given the above I think the best next thing to do is to get in touch with Babel Color for clarification on usage.

[2] is clear on the usage of X-Rite data though: 

>  This information can be used for personal and educational but not for commercial purposes without license from X-rite, Inc.

References
---
1. BabelColor. (2012). ColorChecker RGB and spectra. Retrieved from http://www.babelcolor.com/download/ColorChecker_RGB_and_spectra.xls
2. http://xritephoto.com/ph_product_overview.aspx?ID=1257&Action=Support&SupportID=5159
3. http://xritephoto.com/ph_product_overview.aspx?ID=938&Action=Support&SupportID=5884

**Comment by thomasmansencal** [comment]
@peteroupc: Danny Pascale from Babel Color granted us permission to use the data of [1].

I sent a mail to X-Rite regarding commercial usage of [3], I'll reply to the thread whenever I receive their answer.

**Comment by thomasmansencal** [comment]
Still haven't received an answer from X-Rite. I'll send another mail.

**Comment by thomasmansencal** [comment]
No answers from X-Rite so far, I'll close that issue for now and re-open whenever I hear from them.
